source/Halibut/Transport/Protocol/MessageExchangeStream.cs

```diff
@@ -143,10 +149,7 @@ async Task<string> ReadLineAsync()
 
         public void IdentifyAsSubscriber(string subscriptionId)
         {
-            streamWriter.Write("MX-SUBSCRIBER ");
-            streamWriter.Write(currentVersion);
-            streamWriter.Write(" ");
-            streamWriter.Write(subscriptionId);
+            streamWriter.Write($"{MxSubscriber} {currentVersion} {subscriptionId}");
```

Comment:
I wonder if we should make this a `WriteLine` instead? (and drop the `streamWriter.WriteLine();` on the next line)

Reply:
Agreed. The next PR targets this area so I'll do it as part of that PR.